gcp/appengine/frontend_handlers.py

```diff
@@ -148,8 +148,9 @@ def blog_post_static_files(blog_name: str, file_name: str):
 
 
 @blueprint.route('/about')
```

Comment:
Might be better to just have `/about` be a redirect rather than serving the same page. It makes it slightly confusing to the user, and also is unclear what url `url_for('frontend_handlers.faq')` will return (though in this case I think it will always return the bottom route (faq))

Reply:
+1. redirecting makes it clearer that there is a new canonical link for this.

Reply:
Ack, I believe I have done this correctly, PTAL.